composer.json

```diff
@@ -26,10 +32,11 @@
         "phpstan/phpstan-phpunit": "^1.0",
         "phpstan/phpstan-symfony": "^1.0",
         "phpunit/phpunit": "^9.6",
-        "symfony/browser-kit": "^4.4 || ^5.0 || ^6.0",
-        "symfony/console": "^4.4 || ^5.0 || ^6.0",
-        "symfony/dotenv": "^4.4 || ^5.0 || ^6.0",
-        "symfony/error-handler": "^4.4 || ^5.0 || ^6.0",
+        "symfony/browser-kit": "^4.4 || ^5.4 || ^6.3 || ^7.0",
+        "symfony/console": "^4.4 || ^5.4 || ^6.3 || ^7.0",
+        "symfony/dotenv": "^4.4 || ^5.4 || ^6.3 || ^7.0",
+        "symfony/error-handler": "^4.4 || ^5.4 || ^6.3 || ^7.0",
+        "symfony/security-bundle": "^4.4 || ^5.4 || ^6.3 || ^7.0",
```

Comment:
the security bundle did not define its dependency correctly in Symfony 6.0 to avoid mixed installation of unsupported packages we force specific versions here to avoid: https://github.com/sulu/SuluHeadlessBundle/actions/runs/9448421512/job/26022383599?pr=139

Reply:
This will be fixed in https://github.com/sulu/SuluCommentBundle/pull/55 and so the change can may be reverted